### PR TITLE
Remove mobile site

### PR DESCRIPTION
**Pull request by veidenberg**
## Description

Clean up code, configurations and css styles used in mobile site.
Corresponding PRs in other repos: [ebi-plugins](https://github.com/Ensembl/ebi-plugins/pull/124/), [public-plugins](https://github.com/Ensembl/public-plugins/pull/557), [eg-web-common](https://github.com/EnsemblGenomes/eg-web-common/pull/99), [eg-web-bacteria](https://github.com/EnsemblGenomes/eg-web-bacteria/pull/6), [eg-web-parasite](https://github.com/EnsemblGenomes/eg-web-parasite/pull/7).

## Views affected

All views when visiting the site with a mobile device.
Sandbox: http://wp-np2-1d.ebi.ac.uk:1630 (www with e107 code and e106 data).

## Possible complications

Some web page elements might break or not look right when mobile code dependecies are not removed cleanly or if the default view is not optimized well for small screens.

## Related JIRA Issues (EBI developers only)

https://www.ebi.ac.uk/panda/jira/browse/ENSWEB-6598

### CODE REVIEW

File: modules/EnsEMBL/Web/Component/Shared.pm [hunk 1]
```diff
@@ -25,712 +25,12 @@ package EnsEMBL::Web::Component::Shared;
 use strict;
 
 use HTML::Entities  qw(encode_entities);
-use Text::Wrap      qw(wrap);
```
Comment (ens-ap5) [inline modules/EnsEMBL/Web/Component/Shared.pm:28]: I don't think your branch is up-to-date - there are a load of changes here that are from my code refactoring, and are nothing to do with the mobile site.

**Review by ens-ap5 — CHANGES_REQUESTED** [review]
Mostly looking good apart from a couple of files, as commented.

File: conf/perl.startup [hunk 2]
```diff
@@ -240,7 +240,6 @@ if (Apache2::ServerUtil::restart_count == 1) { # Only parse the config on first
   $conf->ENSEMBL_PROXY_PROTOCOL, $conf->ENSEMBL_BASE_URL,
   $conf->ENSEMBL_PROXY_PROTOCOL, $conf->ENSEMBL_SERVER,     $conf->ENSEMBL_PORT,
   $conf->ENSEMBL_NGINX_PORT ? sprintf("\n NGINX Address:      %s://%s:%s", $conf->ENSEMBL_PROXY_PROTOCOL, $conf->ENSEMBL_SERVER, $conf->ENSEMBL_NGINX_PORT) : '',
-  $conf->MOBILE_URL ? sprintf("\n Mobile URL:         %s", $conf->MOBILE_URL) : '',
```
[the remark this answers is not in the document]
Comment (veidenberg) [reply conf/perl.startup:243]: Sorry, which %s? The whole line is deleted so sprintf %s goes with it.

File: modules/EnsEMBL/Web/Component/Shared.pm (hunk 1 above)
Comment (veidenberg) [reply modules/EnsEMBL/Web/Component/Shared.pm:28]: Yes, I have forgotten to update the base branch before branching. I'll recreate the branch and the PR.

**Comment by veidenberg** [comment]
New PR with updated base branch: https://github.com/Ensembl/ensembl-webcode/pull/899

**Pull request closed without merging**